src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/maui.aotprofile.txt

```diff
@@ -4865,10 +4866,10 @@ Methods:
 	void Microsoft.Maui.Controls.Frame/<>c:.cctor ()
 	void Microsoft.Maui.Controls.Frame/<>c:.ctor ()
-	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding1138676086 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, Microsoft.Maui.Controls.ImageSource>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
-	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding114022620 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, string>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
-	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding1635156915 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, Microsoft.Maui.Controls.ImageSource>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
-	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding570983195 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, string>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
+	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding1310226827 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, Microsoft.Maui.Controls.ImageSource>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
+	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding1361557553 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, string>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
+	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding1463606174 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, Microsoft.Maui.Controls.ImageSource>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
+	void Microsoft.Maui.Controls.Generated.GeneratedBindingInterceptors:SetBinding57528121 (Microsoft.Maui.Controls.BindableObject,Microsoft.Maui.Controls.BindableProperty,System.Func`2<Microsoft.Maui.Controls.BaseShellItem, string>,Microsoft.Maui.Controls.BindingMode,Microsoft.Maui.Controls.IValueConverter,object,string,object,object,object)
```

Comment:
Just curious, @simonrozsival @jkurdek are the names like `SetBinding1310226827` deterministic? Is it a hash of some kind?

Reply:
Yes, its a hash of the location

https://github.com/dotnet/maui/blob/d541b54292554f270919e1c491fae7618a58ea62/src/Controls/src/BindingSourceGen/BindingSourceGenerator.cs#L42-L49